libs/live-network/src/network.ts

```diff
@@ -99,15 +99,6 @@ export const errorInterceptor = (error: InterceptedError): InterceptedError => {
 axios.interceptors.request.use(requestInterceptor);
 axios.interceptors.response.use(responseInterceptor, errorInterceptor);
 
-// not react native
-if (!(typeof navigator !== "undefined" && navigator.product === "ReactNative")) {
-  // the keepAlive is necessary when we make a lot of request in in parallel, especially for bitcoin sync. Otherwise, it may raise "connect ETIMEDOUT" error
-  // refer to https://stackoverflow.com/questions/63064393/getting-axios-error-connect-etimedout-when-making-high-volume-of-calls
-  // eslint-disable-next-line global-require,@typescript-eslint/no-var-requires
-  const https = require("https");
-  axios.defaults.httpsAgent = new https.Agent({ keepAlive: true });
```

Comment:
I think you shouldn't remove this code. You just need to replace the condition 
```
if (!(typeof navigator !== "undefined" && navigator.product === "ReactNative"))
```

we should run this piece of code for LLD

Reply:
if you remove this piece of code, it raises an "connect ETIMEDOUT" error in windows if I remember well

Reply:
Not sure to understand why removing this code would cause a timeout on windows ? The condition is only executed of we're on ReactNative